### ISSUE TITLE
Versions of quick-xml > 0.26 give unexpected parse error Custom("missing field `Name`")

### ISSUE THREAD

**Issue by ChrisWhealy** [tafia/quick-xml#656]
The XML I'm working with parses find in version of `quick-xml` up to and including `0.26`, but as soon as I move to version `0.27` or higher (and without changing anything else in the coding) I get an unexpected error:

```
Custom("missing field `Name`")
```

The easiest way to illustrate this is by a simple example I've put into this repo https://github.com/ChrisWhealy/quick-xml-parse-error

In `Cargo.toml`, if `quick-xml` version is set to `0.26`, `cargo test` works fine.

Change the `quick-xml` version to `0.27` or higher, and the test no fails.

Is this a bug or have I missed something?

**Comment by Mingun** [comment]
Yes, quick-xml 0.27 changes the way how attributes are handled, not they should be explicitly marked in structs by prepending a `@` symbol to their names. Rename your attribute fields to follow that.

Duplicate of #534 and #543
